### DISCUSSION TITLE
Deployment of github page failing

### DISCUSSION THREAD

**Discussion by anyachandorkar** [community/community#49971] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am using the jekyll template Architect following instructions here: https://github.com/pages-themes/architect

While I got it to deploy the first time with just the header, everytime I add an index.md file to add text to my homepage, the deployment fails. Right now my website looks like this: https://anyachandorkar.github.io/servicechain.io/

But I want to add text to the body.

**Comment by radar-blip** [comment]
You should be adding your text to the existing index.md, 
no need to create additional index.md files.
